src/training/day3/Homework.java

```diff
@@ -0,0 +1,46 @@
+package training.day3;
+
+import java.util.Scanner;
+
+public class Homework {
+
+    public static boolean checker(int number){
+        boolean flag = false;
+        if (number % 2 == 0){
+            flag = true;
+        }
+        return flag;
+    }
+    public static int smallest(int num1, int num2, int num3){
+        int highest = 0;
+        if (num1 < num2 && num2 < num3){
+            highest = num1;
+            return highest;
+        } else if (num2 < num1 && num1 < num3) {
+            highest = num2;
+        } else if (num3 < num2 && num2 < num1) {
+            highest = num3;
+        }
+        return highest;
+    }
```

Comment:
New line after the end of the method. And every method, getter, setter, constructor etc.